toolchain/bootstrap/modules.sh

```
@@ -118,11 +118,31 @@ if [ $(echo "$VARIABLES" | grep = | wc -c) -gt 0 ]; then
 fi
 
 # Don't check for Cray paths on Carpenter, otherwise do check if they exist
-if [ ! -z ${CRAY_LD_LIBRARY_PATH+x} ] && [ "$u_c" '!=' 'c' ]; then
+if [ ! -z ${CRAY_LD_LIBRARY_PATH+x} ] && [ "$u_c" '!=' 'c' ] &&  [ "$u_c" '!=' 'famd' ] ; then
     ok "Found $M\$CRAY_LD_LIBRARY_PATH$CR. Prepending to $M\$LD_LIBRARY_PATH$CR."
     export LD_LIBRARY_PATH="$CRAY_LD_LIBRARY_PATH:$LD_LIBRARY_PATH"
 fi
 
+if[ "$u_c" '==' 'famd' ]; then 
```

Shellcheck (0.11.0)

[error] You need a space before the [.

(SC1069)